app/views/admin/users/reports/_index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2 class="crayons-subtitle-1">Reports submitted by @<%= @user.username %> (<%= @related_reports.count %>)</h2>
+
+<% unless @related_reports.empty? %>
+  <div class="pt-3">
+    <% @related_reports.each do |report| %>
+      <article class="c-list-item c-list-item--interactive flex justify-between gap-4">
+        <div>
+          <header class="flex gap-3 items-center">
+            <h3 class="crayons-subtitle-3">
+              <a href="<%= admin_report_path(report.id) %>" class="c-link">
+                <%= report.category.capitalize %>
+              </a>
+            </h3>
+            <span class="c-indicator c-indicator--relaxed  <%= "c-indicator--danger" if report.status.casecmp("open").zero? %> <%= "c-indicator--success" if report.status.casecmp("resolved").zero? %>">
```

Comment:
1. This should probably become a helper
2. @Ridhwana did mention we should be careful with letters capitalization here... 🤷  Just pointing it out.

Reply:
Will tackle in a subsequent PR.